cdk/PgStacInfra.ts

```diff
@@ -458,6 +462,57 @@ export class PgStacInfra extends Stack {
         }),
       );
     }
+
+    if (itemGenConfig) {
+      // item loader
+      const stacItemLoader = new StacItemLoader(this, "stac-item-loader", {
+        pgstacDb,
+        vpc: vpc,
+        subnetSelection: apiSubnetSelection,
+        batchSize: 500,
+        lambdaTimeoutSeconds: 300,
+        environment: {
+          CREATE_COLLECTIONS_IF_MISSING: "TRUE",
```

Comment:
This makes it so a boilerplate collection is generated and uploaded before an item if the collection does not exist yet.